<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdatePassengerModal</class>
 <widget class="QWidget" name="UpdatePassengerModal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>308</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font-weight: 600; font-size: 10pt; color: black;</string>
     </property>
     <property name="text">
      <string>Пассажир</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,0">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="formLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <property name="horizontalSpacing">
          <number>40</number>
         </property>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="inputGroup_11">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="pSerialNoLabel">
             <property name="text">
              <string>Серийный номер паспорта</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="pSerialNoInput">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="minimum">
              <number>1000</number>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
             <property name="value">
              <number>9999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QVBoxLayout" name="inputGroup_10">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="pCodeLabel">
             <property name="text">
              <string>Номер паспорта</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="pCodeInput">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="minimum">
              <number>100000</number>
             </property>
             <property name="maximum">
              <number>999999</number>
             </property>
             <property name="value">
              <number>999999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="inputGroup_6">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="lastNameLabel">
             <property name="text">
              <string>Фамилия</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lastNameInput">
             <property name="echoMode">
              <enum>QLineEdit::Normal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="inputGroup_9">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="firstNameLabel">
             <property name="text">
              <string>Имя</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="firstNameInput">
             <property name="echoMode">
              <enum>QLineEdit::Normal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QVBoxLayout" name="inputGroup_12">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="middleNameLabel">
             <property name="text">
              <string>Отчество</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="middleNameInput">
             <property name="echoMode">
              <enum>QLineEdit::Normal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Сохранить изменения</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
